pom.xml

```diff
@@ -279,6 +279,12 @@
         <artifactId>gson</artifactId>
         <version>2.8.0</version>
     </dependency>
+
+    <dependency>
```

Comment:
@dan144 @dbangera23 What does this do if tesseract isn't installed? This is probably something best solved with a detailed and intelligent error message.